ddl/sequence.go

```diff
@@ -160,6 +160,7 @@ func buildSequenceInfo(stmt *ast.CreateSequenceStmt, ident ast.Ident) (*model.Se
 		switch op.Tp {
 		case ast.TableOptionComment:
 			sequenceInfo.Comment = op.StrValue
+		case ast.TableOptionEngine:
```

Comment:
What does this mean?

Reply:
Em... compatible with `create sequence seq Engine=InnoDB`, otherwise it will cast `Unsupported sequence table-option InnoDB`.

Reply:
It'll be better to add a comment for this

Reply:
ok!